operatorcourier/verified_manifest.py

```diff
@@ -20,81 +95,70 @@ def bundle(self):
         return format_bundle(self.bundle_dict)
 
     def __init__(self, source_dir, yamls, ui_validate_io, repository):
+        self.nested = False
```

Comment:
Is this needed? Shouldn't this boolean be defined as `False` by default?

Reply:
@kevinrizza I think it is needed because unlike Java, Python does not have a way to explicitly declare a variable with a type that has a default value.

If I remove this line, my IDE will give a warning at this [line](https://github.com/operator-framework/operator-courier/pull/122/files#diff-9bab59f69ef6c22e41f4c0d543b8e228R138) where the boolean is used, saying **instance attribute defined outside \_\_init\_\_**. I guess that's a bad practice in Python?

Reply:
Good point.